python/adelphi/README.md

```diff
@@ -0,0 +1,107 @@
+
+    adelphi --keyspaces foo,bar contribute
+
+Authentication to Github is performed by way of a [personal access token](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token).  You must create a token for your Github user before you can contribute your schema(s) to Adelphi.  The token can be provided to the command at execution time using a command-line argument but this is discouraged for security reasons.  Instead we recommend using an environment variable, in this case the **ADELPHI_CONTRIBUTE_TOKEN** environment variable.  We discuss using environment variables to pass command-line arguments in the "Options" section below.
+
+## Options
+The "adelphi" application supports several command-line arguments.  The full list of arguments can be accessed via the following:
+
+    adelphi --help
+
+The output of this command provides a brief summary of each argument:
+
+    $ adelphi --help
+    Usage: adelphi [OPTIONS] COMMAND [ARGS]...
+    
+    Options:
+      --hosts TEXT                  Comma-separated list of contact points
+                                    [default: 127.0.0.1]
+    
+      --port INTEGER                Database RPC port  [default: 9042]
+      --username TEXT               Database username
+      --password TEXT               Database password
+      --keyspaces TEXT              Comma-separated list of keyspaces to include.
+                                    If not specified all non-system keypaces will
+                                    be included
+    
+      --rf INTEGER                  Replication factor to override original
+                                    setting. Optional.
+    
+      --anonymize / --no-anonymize  Enable/disable schema anonymization  [default:
+                                    True]
+    
+      --output-dir TEXT             Directory schema files should be written to.
+                                    If not specified, it will write to stdout
+    
+      --purpose TEXT                Comments on the anticipated purpose of this
+                                    schema.  Optional.
+    
+      --maturity TEXT               The maturity of this schema.  Sample values
+                                    would include 'alpha', 'beta', 'dev', 'test'
+                                    or 'prod'.  Optional.
+    
+      --help                        Show this message and exit.
+    
+    Commands:
+      contribute     Contribute schemas to Adelphi
+      export-cql     Export a schema as raw CQL statements
+      export-gemini  Export a schema in a format suitable for use with the the...
+
+Individual commands may have their own options and/or help text.  For example the help for the "contribute" command is as follows:
+
+    $ adelphi contribute --help  
+    Usage: adelphi contribute [OPTIONS]
```

Comment:
that `click` lib makes this a little weird, sometimes options come before, other times after. I understand it gives some positional context to the args, but I'd prefer to pass the `--aaa`  options at whatever position and it would still handle it. Not related to this PR of course, but we can probably check later if we can be more flexible here.

Reply:
I see your point... kinda.  I personally like the fact that the position of the args is used to be very explicit about what is command-specific and what is just used as part of running the app.

I _believe_ click supports the ability for command declarations to "inherit" options from their parent container (or something along those lines... I forget exactly how it works now).  Upshot is that all the args which are currently not tied to a command would devolve into command-specific args, which I believe would accomplish what you're describing.  I guess I'm not completely opposed to that; I even experimented with something like that when click first was brought in.  I personally still like the clear demarcation of arg types but I readily agree this may be an idiosyncrasy about me. :)